infrastructure/worker/job.go

```diff
@@ -0,0 +1,25 @@
+package worker
+
+type Job interface {
+	Apply() bool
+	GetData() int
+}
+
+type NewsJobFilter struct {
+	data int
+}
+
+func NewNewsJobFilter(data int) Job {
+	return &NewsJobFilter{data: data}
+}
+
+func (n NewsJobFilter) Apply() bool {
+	if n.data%2 == 0 {
```

Comment:
You can just do `return n.data %2 == 0`

Reply:
This is an outdated version of the file, please check the current one.